### DISCUSSION TITLE
printer.cfg overwritten with default version every shutdown/restart

### DISCUSSION THREAD

**Discussion by fbson** [Guilouz/Creality-Helper-Script-Wiki#593] [Issues]
Hello.
I'm finding that any changes made that get written to the printer.cfg file are subsequently wiped out as they don't survive a power off / shutdown. After each power on the printer.cfg is back to the original/default one, with what should have remained as the printer.cfg file now being created as a new 'printer.cfg.1' file in the same location. This is the case for me on two K1C printers.
Is there anything obvious that I'm doing wrong?..!
Thanks!

**Comment by Guilouz** [comment]
Read the wiki^^

https://guilouz.github.io/Creality-Helper-Script-Wiki/helper-script/prevent-and-allow-updating-klipper-configuration-files/

**Reply by fbson** [reply]
Thanks for the quick reply. Apologies too if it is that basic, I'll give that a try and shutdown in an hour or two once the current prints have finished.
Thanks again for your work, brlliant!

**Reply by fbson** [reply]
Having looked back at it, it's not that though - updating Klipper configuration files was already allowed. Any other thoughts?
Thanks.

**Reply by fbson** [reply]
Unless I'm being really dim, and having re-read that, I need to prevent...!!

**Reply by fbson** [reply]
Ha, that'll be it then - as I guess you already knew...!!
My only complaint then is that your original reply shouldn't have been 'read the wiki', it should have said 'don't misread the wiki'...!!
Apologies and thanks!
